sdks/go/pkg/beam/core/graph/coder/coder.go

```diff
@@ -126,8 +126,17 @@ const (
 	VarInt        Kind = "varint"
 	WindowedValue Kind = "W"
 	KV            Kind = "KV"
-	GBK           Kind = "GBK"
-	CoGBK         Kind = "CoGBK"
+
+	// CoGBK is non-standard and currently equivalent to either
```

Comment:
Please clarify this comment. I assume this comment to mean that CoGBK is not currently a top-level Beam primitive (JIRA pending), but if that's the case, please say it like that. If it's not the case, I don't know what non-standard means, so it needs explaining as well.